### PR TITLE
Workaround for titlebar drag area on windows 10

### PR DESCRIPTION
**Pull request by gave92**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Fixes dragging the window on windows 10

Despite the docs, SetDragRectangles() still does not work on win10 with winappsdk "1.2.220902.1-preview1" :/
Confirmed on Discord.
The workaround is kinda ugly, if you prefer to just wait for a fix from winappsdk I'm ok to not merge this. Does not change things on win11.

**Validation**
How did you test these changes?
- [x] Built and ran the app

### CODE REVIEW

**Comment by yaichenbaum** [comment]
Let's wait for WinAppSdk to fix this.

**Review by yaichenbaum — APPROVED** [review]
LGTM

File: src/Files.App/Views/MainPage.xaml.cs [hunk 1]
```diff
@@ -134,7 +134,18 @@ private void UserSettingsService_OnSettingChangedEvent(object sender, SettingCha
...
-			horizontalMultitaskingControl.DragArea.SizeChanged += (_, _) => SetRectDragRegion();
+			// WINUI3: bad workaround to be removed asap!
+			// SetDragRectangles() does not work on windows 10 with winappsdk "1.2.220902.1-preview1"
+			if (Environment.OSVersion.Version.Build >= 22000)
+			{
+				horizontalMultitaskingControl.DragArea.SizeChanged += (_, _) => SetRectDragRegion();
+			}
+			else
+			{
+				App.Window.AppWindow.TitleBar.ExtendsContentIntoTitleBar = false;
```
Comment (d2dyno1) [inline src/Files.App/Views/MainPage.xaml.cs:145]: `App.Window.AppWindow.TitleBar` is null on Windows 10

Comment (gave92) [reply src/Files.App/Views/MainPage.xaml.cs:145]: Not anymore (appsdk 1.2 preview1)

**Pull request merged**